### DISCUSSION TITLE
Site is not loading (See screenshsots)

### DISCUSSION THREAD

**Discussion by WeredokOff** [vercel/community#2085] [Help]
### Summary

![изображение](https://user-images.githubusercontent.com/128225125/232209974-4b285aef-f646-4d89-94a6-92c50f6e2682.png)
![изображение](https://user-images.githubusercontent.com/128225125/232209981-54ec5d66-0f7f-4967-98d2-a70421d82689.png)
![изображение](https://user-images.githubusercontent.com/128225125/232209988-ad444427-ea76-4cc2-ab37-44ae0cd58cf1.png)

Me waiting more than 15 minutes, and it's still not loaded

### Example

vercel.com

**Comment by WeredokOff** [comment]
after i cleared all cookies for vercel.com and vercel.live i can't login to my account:
![изображение](https://user-images.githubusercontent.com/128225125/232210331-fe39fb52-b516-495b-b21c-79d493b30c9e.png)

i can't put anywhere my login data, and i can't click any button...

**Comment by WeredokOff** [comment]
vercel.com/new looks like this
![изображение](https://user-images.githubusercontent.com/128225125/232210636-f694a9cc-fa91-4619-b33c-7e4f7851d4dc.png)

**Comment by swarnava** [comment]
Hi,

Are you still facing this issue?

**Reply by Weredok** [reply]
yes

**Reply by WeredokOff** [reply]
![изображение](https://user-images.githubusercontent.com/128225125/233693117-409b5b2d-fb10-4592-b9c3-6e75df07fd1b.png)
still...

**Reply by WeredokOff** [reply]
> Hi,
> 
> Are you still facing this issue?

2 weeks... When maintainers will fix it?

**Reply by amyegan** [reply]
Do you see any error information in the dev tools console or network tab? I'm not able to reproduce this problem

**Comment by maddsua** [comment]
> ### Summary
> 
> ![изображение](https://user-images.githubusercontent.com/128225125/232209974-4b285aef-f646-4d89-94a6-92c50f6e2682.png) ![изображение](https://user-images.githubusercontent.com/128225125/232209981-54ec5d66-0f7f-4967-98d2-a70421d82689.png) ![изображение](https://user-images.githubusercontent.com/128225125/232209988-ad444427-ea76-4cc2-ab37-44ae0cd58cf1.png)
> 
> Me waiting more than 15 minutes, and it's still not loaded
> ### Additional information
> 
> _No response_
> ### Example
> 
> vercel.com

What's the Firefox version? I don't seem to have the issue on 102.10.0esr

**Comment by maddsua** [comment]
Firefox sometimes can "get wasted" which results in weird behavior. A stupid suggestion, but try to clear entire browser cache and restart ALL Firefox instances
